legend-engine-xt-serviceStore-grammar/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/ServiceStoreLexerGrammar.g4

```diff
@@ -2,6 +2,13 @@ lexer grammar ServiceStoreLexerGrammar;
 
 import M3LexerGrammar;
 
+@lexer::members{
```

Comment:
this seems too complex for this. Can you please review how we do this for RelationalDatabaseConnection/ Relational milestoning and do it similarly